_topic_map.yml

```diff
@@ -51,7 +51,7 @@ Dir: architecture
 Distros: openshift-acs
 Topics:
 - Name: Product architecture
-  File: index
+  File: acs-architecture
```

Comment:
Why?

Reply:
It was a recommendation to not to have same file names for assemblies. We already have a index file in welcome dir.